mlos_bench/mlos_bench/tests/environments/remote/conftest.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Fixtures for the RemoteEnv tests using SSH Services.
+"""
+
+import mlos_bench.tests.services.remote.ssh.fixtures as ssh_fixtures
+
+# Expose those as local names.
+ssh_test_server_hostname = ssh_fixtures.ssh_test_server_hostname
+ssh_test_server = ssh_fixtures.ssh_test_server
+alt_test_server = ssh_fixtures.alt_test_server
+ssh_host_service = ssh_fixtures.ssh_host_service
+ssh_fileshare_service = ssh_fixtures.ssh_fileshare_service
```

Comment:
can we just do
```suggestion
from mlos_bench.tests.services.remote.ssh.fixtures import (
    ssh_test_server_hostname,
    ssh_test_server,
    alt_test_server,
    ssh_host_service,
    ssh_fileshare_service,
)
```
here or maybe in the test file that actually uses them?

Reply:
It's used in several files - the `conftest.py` method was easier and followed the usual pattern of keeping fixtures discoverable there for the most part.

Reply:
Leaving as is for now.